docs/DeveloperGuide.md

```diff
@@ -791,14 +758,14 @@ Priorities: High (must have) - `* * *`, Medium (nice to have) - `* *`, Low (unli
 
   Use case resumes at step 2.
 
-**Use case: Delete a patient's appointment**
+<h3>Use case 5: Removing a patient's next appointment date</h3>
```

Comment:
This use case was for deleting past appointments but I think the new one works fine also, just an FYI

Reply:
Changed deliberately to make Use Cases sound more relatable to an actual event. Thanks for heads up!